index.html

```diff
@@ -12,17 +12,18 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <base href="/">
```

Comment:
Worth noting that base can be set once per document. It's usage is typically considered esoteric, but we will be co-opting it for our purposes, possibly to the exclusion of someone else's.